src/components/CorpusIndex.js

```diff
@@ -153,20 +153,22 @@ const CorpusIndex = ({data}) => {
         {
           props => (
             <div>
-              <SearchBar {...props.searchProps}/>
-              {(data.description || data.license) && (
-                <div>
+                <div className="corpus-description">
+                {(data.description || data.license) && (
+                  <div>
                   {data.description && (
                     <ReactMarkdown>{data.description}</ReactMarkdown>
                   )}
                   {data.licence && (
                     <p>
                       <span>Corpus licensed under </span>
-                      <a href={data.licenceUrl}>{data.licence}</a>.
+                      <a target="_blank" rel="noopener noreferrer" className={data.licence} href={data.licenceUrl}>{data.licence}</a>
```

Comment:
We cannot use the license as a class name because this can be random text not necessarily meeting the class name requirements. The licence can be identified by URL though. I'll change that.

Reply:
Absolutely. Your solution makes much more sense. Thank you for suggesting it! 
Are there actually more licenses that we should consider adding? Up until now I've only seen two kinds of licenses (CC0 & CC-BY-NC) for the current corpora.

Reply:
At the moment we have only CC0 and CC-BY-NC. Once other corpora get license information we can successively add more icons if needed.